build.rs

```diff
@@ -80,7 +80,7 @@ fn build_nasm_files() {
   config_file.write(b"	%define PIC 1\n").unwrap();
   config_file.write(b" %define STACK_ALIGNMENT 16\n").unwrap();
   config_file.write(b" %define HAVE_AVX512ICL 0\n").unwrap();
-  if cfg!(target_os = "macos") {
```

Comment:
is the target_os cfg deprecated now? did we have documentation for it?

Reply:
It is not deprecated, it is the build-time information for build.rs, so it is always the host os.
to have the target_os for what we are building for you have to use the env variable.